connectivity/drivers/lora/TARGET_STM32WL/STM32WL_LoRaRadio.cpp

```diff
@@ -202,6 +206,22 @@ void STM32WL_LoRaRadio::rf_irq_task(void)
 }
 #endif
 
+/**
+  * @brief This function handles SUBGHZ Radio Interrupt.
+  */
+static void SUBGHZ_Radio_IRQHandler(void)
+{
+#ifdef MBED_CONF_RTOS_PRESENT
+  
+    irq_thread.flags_set(SIG_INTERRUPT);
+  
+    irq_status_rtos = (radio_irq_masks_t)STM32WL_LoRaRadio::get_irq_status();
```

Comment:
Even in case of using a thread, I would tend to keep reading this status only in one place : in RadioIrqProcess. Otherwise there is a risk of this value is modified in this IRQ handler, while it is being use din the thread.
So either only access it in one place (my first recommendation) or protect it against multiple access

Reply:
> @LMESTM could you have a look ?

I had a look and don't see reasons for a crash. But I think there is a possible race condition that should be avoided so I added a comment for @ludoch-stm to look at